docs/source/linalg.rst

```diff
@@ -80,6 +80,7 @@ Tensor Operations
 
     tensorinv
     tensorsolve
```

Comment:
It'd be good also to update the docs for `tensorinv` and `tensorsolve` to point to `linalg.tensordot` rather than `torch.tensordot`.

Reply:
I'm confused: isn't it exactly what I did in commit 0d17d668f9?

Reply:
Ah, yes, sorry, I missed that!